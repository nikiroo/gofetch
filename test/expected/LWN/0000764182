                   SECURITY UPDATES FOR WEDNESDAY                  

  [Security] Sep 5, 2018 15:01 UTC (Wed) (ris)

  o News link: https://lwn.net/Articles/764182
  o Source link: 


    Dist.
    
    ID
    
    Release
    
    Package
    
    Date
    
    Debian
    
    [1]DSA-4284-1
    
    stable
    
    lcms2
    
    2018-09-04
    
    openSUSE
    
    [2]openSUSE-SU-2018:2623-1
    
    42.3
    
    yubico-piv-tool
    
    2018-09-05
    
    Oracle
    
    [3]ELSA-2018-4208
    
    OL6
    
    kernel
    
    2018-09-04
    
    Oracle
    
    [4]ELSA-2018-4208
    
    OL7
    
    kernel
    
    2018-09-04
    
    SUSE
    
    [5]SUSE-SU-2018:2608-1
    
    cobbler
    
    2018-09-04
    
    SUSE
    
    [6]SUSE-SU-2018:2615-1
    
    SLE11
    
    kvm
    
    2018-09-05
    
    
    
    [1] https://lwn.net/Articles/764176/
    
    [2] https://lwn.net/Articles/764177/
    
    [3] https://lwn.net/Articles/764178/
    
    [4] https://lwn.net/Articles/764179/
    
    [5] https://lwn.net/Articles/764180/
    
    [6] https://lwn.net/Articles/764181/



