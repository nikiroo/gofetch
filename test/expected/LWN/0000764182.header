0Security updates for Wednesday	null/LWN/0000764182		70
i  [Security] Sep 5, 2018 15:01 UTC (Wed) (ris)
i
i    Security  updates  have been issued by Debian (lcms2), openSUSE
i    (yubico-piv-tool),  Oracle  (kernel),  and  SUSE  (cobbler  and
i    kvm).
i
