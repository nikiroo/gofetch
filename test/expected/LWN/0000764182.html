<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Security updates for Wednesday</h2>
	<div class='details'>([Security] Sep 5, 2018 15:01 UTC (Wed) (ris))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000764182</a></li>
        <li>News link: <a href='https://lwn.net/Articles/764182'>https://lwn.net/Articles/764182</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Dist.<br/><br/>ID<br/><br/>Release<br/><br/>Package<br/><br/>Date<br/><br/>Debian<br/><br/>[1]DSA-4284-1<br/><br/>stable<br/><br/>lcms2<br/><br/>2018-09-04<br/><br/>openSUSE<br/><br/>[2]openSUSE-SU-2018:2623-1<br/><br/>42.3<br/><br/>yubico-piv-tool<br/><br/>2018-09-05<br/><br/>Oracle<br/><br/>[3]ELSA-2018-4208<br/><br/>OL6<br/><br/>kernel<br/><br/>2018-09-04<br/><br/>Oracle<br/><br/>[4]ELSA-2018-4208<br/><br/>OL7<br/><br/>kernel<br/><br/>2018-09-04<br/><br/>SUSE<br/><br/>[5]SUSE-SU-2018:2608-1<br/><br/>cobbler<br/><br/>2018-09-04<br/><br/>SUSE<br/><br/>[6]SUSE-SU-2018:2615-1<br/><br/>SLE11<br/><br/>kvm<br/><br/>2018-09-05<br/><br/><br/><br/>[1] https://lwn.net/Articles/764176/<br/><br/>[2] https://lwn.net/Articles/764177/<br/><br/>[3] https://lwn.net/Articles/764178/<br/><br/>[4] https://lwn.net/Articles/764179/<br/><br/>[5] https://lwn.net/Articles/764180/<br/><br/>[6] https://lwn.net/Articles/764181/
	</div>
<hr/>
</div>
</body>
